docs/_includes/css/forms.html

```diff
@@ -976,13 +988,16 @@ <h2 id="forms-control-sizes">Control sizing</h2>
     <form>
       <div class="row">
         <div class="col-xs-2">
-          <input type="text" class="form-control" placeholder=".col-xs-2">
+          <label for="col-size-1"></label>
+          <input type="text" class="form-control" id="col-size-1" placeholder=".col-xs-2">
```

Comment:
if you're adding empty ``<label ...></label>`` constructs, you're really just coding to please a validation tool here. as mentioned above, the ``placeholder`` on the inputs is a valid (though perhaps not pretty) way to provide an accessible name. using this empty label construct, you're actually overriding it and injecting an empty/null label to the input, neutering it...